readme.md

```diff
@@ -232,7 +232,8 @@ The most common reasons for Basset to fail are:
 Ensure that APP_URL in your `.env` matches your server configuration, including the hostname, protocol, and port number. Incorrect settings can lead to asset loading issues.
 
 2) **Improperly configured disk.**  
-By default, Basset uses the public disk since its files must be public. For new Laravel projects, the configuration is usually correct. If you're upgrading from an older version, check `config/backpack/basset.php`. Make sure your `config/filesystems.php` public disk looks like: https://github.com/laravel/laravel/blob/10.x/config/filesystems.php#L39-L45.
+By default, Basset uses the public disk since its files must be public. For new Laravel projects, the configuration is usually correct. If you're upgrading from an older version, check `config/backpack/basset.php`. Make sure your `config/filesystems.php` public disk looks like: https://github.com/laravel/laravel/blob/10.x/config/filesystems.php#L39-L45.  
+If you prefer not to change your public disk, you can create your own disk and set it in the Basset config, or you can use the provided disk named `basset` by setting it in the `config/backpack/basset.php`.
```

Comment:
I already added a comment in the other PR regarding this, maybe we remove this from this PR and make the full readme change in the `check-command` branch.